### PR TITLE
Refresh store page to reflect new store UX more accurately

### LINKED ISSUE #228: Pushing and releasing snaps documentation should exclusively refer to snapcraft
https://docs.snapcraft.io/core/store

This page talks about being able to upload snaps using the dashboard, but this feature is deprecated in favour of the snapcraft command line. It should probably just link to the existing publishing docs at https://docs.snapcraft.io/build-snaps/publish

### PR DESCRIPTION
**Pull request by caldav**
Some clean up.

Fixes #228

### CODE REVIEW

File: core/store.md [hunk 1]
```diff
@@ -3,51 +3,49 @@ layout: base
...
-There are multiple ways to distribute snaps as the format is not tied to a specific distribution system.
+There are multiple ways to distribute snaps as the format is not tied to a specific distribution system. However, the recommended way to distribute snaps is to release them in the Snap Store.
 
 ## The Snap Store
 
-The main way of distributing snaps to snapd systems is through the [Snap Store](https://dashboard.snapcraft.io "Snap Store"), where you can customize how your snap is presented, review each new pushed snap, and control their release process over several release channels. Here is the model it follows.
+The main way of distributing snaps is through the [Snap Store dashboard](https://dashboard.snapcraft.io), where you can customize how snaps are presented, review each new pushed snap, and control the release process over several release channels. Here is the model it follows:
 
-### Developer namespace
+### 1. Developer namespace
```
Comment (evandandrea) [inline core/store.md]: This is called a "Snap Store username" now.

File: core/store.md [hunk 2]
```diff
@@ -3,51 +3,49 @@ layout: base
...
 
-You'll choose a unique developer namespace as part of the account creation process. This namespace will represent you as a publisher in the Snap Store and you won't be able to change it afterwards.
+You'll choose a unique developer namespace as part of the store account creation process. This namespace will represent you as a publisher and you won't be able to change it afterwards.
```
Comment (evandandrea) [inline core/store.md]: What do you think of: "You'll choose a username as part of creating your Snap Store account. This unique name will represent you as a publisher, so pick a name that best reflects your brand. You will not be able to change it once set."

File: core/store.md [hunk 3]
```diff
@@ -3,51 +3,49 @@ layout: base
...
 
-### Naming
+### 2. Naming
 
-You can release a snap under any name you have rights to. Names can be registered by using the `snapcraft register` command, clicking **New Snap** on the developer portal, or by visiting the [Register name](https://dashboard.snapcraft.io/dev/snaps/register-name/ "register name") page. You can also grant other developers permission to release versions of a snap you own, for example as part of an open source project.
+You can release a snap under any name you have rights to. Names can be registered by using the [`snapcraft register`](/build-snaps/register) command, or by visiting the [Register new snap](https://dashboard.snapcraft.io/snaps/register/) dashboard page. You can also grant other developers permission to release versions of a snap you own, for example as part of an open source project.
```
Comment (evandandrea) [inline core/store.md]: This looks good. What do you think of changing the text slightly to: "You can also grant other developer permission to release version of a snap you own on the dashboard "Collaboration" page."? I think "for example as part of an open source project" isn't necessary, but if you want to leave it I'm fine with that.

File: core/store.md [hunk 4]
```diff
@@ -3,51 +3,49 @@ layout: base
...
+Pushing to the Snap Store is done directly from the command-line with the [`snapcraft push`](/build-snaps/upload) command.
 
 It's worth noting that when you push a snap, the Snap Store assigns it a revision number of 1\. The Snap Store then automatically increments this revision number each time you push a new version.
 
-### Releasing
+### 4. Releasing
 
-After you've chosen a channel, your application is sent for review. Most apps are reviewed by way of automated checks, but if your app uses sensitive [interfaces](/core/interfaces), it may be manually reviewed -- you can find more details on the review process [here](https://developer.ubuntu.com/en/publish/application-states/).
+After pushing it, your snap is reviewed by way of automated checks. If your app uses sensitive [interfaces](/core/interfaces), it may be manually reviewed and you will receive an email notifying you of the review state.
```
Comment (evandandrea) [inline core/store.md]: What do you think of: "After pushing your snap, it is reviewed by way automated checks. Most automated reviews pass with no further action needed.

Use of some [interfaces](/core/interfaces) will trigger a manual review. You will receive emails explaining any further action needed and the state of the review process."

File: core/store.md [hunk 5]
```diff
@@ -3,51 +3,49 @@ layout: base
...
 
-Once your snap has been reviewed and approved, you can release it when you're ready, instantly making the snap available to users.
+Once your snap has been reviewed and approved, you can release it using the [`snapcraft release`](/build-snaps/release) command, instantly making the snap available to users.
 
-### Release channels
+#### Release channels
 
-On the Snap Store, snaps can be published into different channels at the same time: `stable`, `candidate` (release candidate), `beta`, and `edge`. This enables you to engage with users who are willing to test changes, and helps users decide how close to the leading edge of development they want to be.
+Snaps can be released into multiple [channels](/reference/channels) (`stable`, `candidate`, `beta`, and `edge`). This enables you to engage with users who are willing to test changes, and helps users decide how close to the leading edge of development they want to be.
```
Comment (evandandrea) [inline core/store.md]: What do you think of: "This enables you to engage with users who are willing to test changes. It lets users decide how close to the leading edge of development they want to be."

File: core/store.md [hunk 6]
```diff
@@ -3,51 +3,49 @@ layout: base
...
 
 By default, snaps are installed from the `stable` channel. Versions of snaps from other channels need to be explicitly selected:
 
-    $ snap install hello --beta
+    $ snap install hello --channel=beta
```
Comment (evandandrea) [inline core/store.md]: I'm fine with this, but what's the motivation? `--beta` does work.

File: core/store.md [hunk 7]
```diff
@@ -3,51 +3,49 @@ layout: base
...
-    hello  (beta) installed
-
-This switches the snap to using this channel for future updates.
+    hello  (edge) installed
 
 ## Brand stores
 
-Brands can also take advantage of white label [brand stores](https://docs.ubuntu.com/core/en/build-store/index), which are branded extensions to the Snap store. Note that this is a commercial offering aimed at companies.
\ No newline at end of file
+Brands can also take advantage of white label [brand stores](https://docs.ubuntu.com/core/en/build-store/index), which are branded extensions to the Snap store. Note that this is a commercial offering aimed at brands.
```
Comment (evandandrea) [inline core/store.md]: What's a brand? 😄

File: core/store.md [hunk 8]
```diff
@@ -3,51 +3,49 @@ layout: base
 title: Snap stores
 ---
 
-There are multiple ways to distribute snaps as the format is not tied to a specific distribution system.
+There are multiple ways to distribute snaps as the format is not tied to a specific distribution system. However, the recommended way to distribute snaps is to release them in the Snap Store.
```
Comment (Greggless) [inline core/store.md]: > However, the best way to distribute snaps is to release them in the Snap Store. Here they can be installed by using the command line for systems that are running [snapd](/core). Once the snap is in the stable channel users can find your snap in the [snap directory](https://snapcraft.io/discover) or by using Gnome Software.

Comment (evandandrea) [inline core/store.md]: This reads a bit defensive. I wonder if we can make it more positive? Something like:

"Given the openness of the snap format, there are many ways to distribute snap applications. Using the Snap Store exposes your application to tens of millions of potential users and gives you detailed install statistics, so using it is recommended."

**Pull request merged**